tests/components/teslemetry/common.py

```diff
@@ -0,0 +1,50 @@
+"""Teslemetry common helpers for tests."""
```

Comment:
The content of this file can just be put in __init.py

Reply:
The irony is I moved it out of __init__ recently

I'll throw it back